src/components/Delegate/StakingOpportunities.tsx

```diff
@@ -73,23 +73,95 @@ export const ValidatorName = ({ moniker, isStaking }: ValidatorNameProps) => {
   )
 }
 
-export const StakingOpportunities = ({ assetId, opportunities }: StakingOpportunitiesProps) => {
-  const isLoaded = true
-  const isStaking = opportunities.some(x => x.cryptoAmount)
-  const assetSymbol = useAppSelector(state => selectAssetByCAIP19(state, assetId)).symbol
+export const StakingOpportunities = ({ assetId }: StakingOpportunitiesProps) => {
+  const stakingDataStatus = useAppSelector(selectStakingDataStatus)
+  const isStakingDataLoaded = stakingDataStatus === 'loaded'
```

Comment:
it would be ideal to do this in a selector, i.e. selectStakingDataIsLoaded

Reply:
Done, and I did the same for the isValidatorLoaded thing